cli/src/semgrep/rule_match.py

```diff
@@ -190,6 +190,10 @@ def get_cli_unique_key(self) -> Tuple:
             self.start.offset,
             self.end.offset,
             self.message,
+            # This is necessary so we don't deduplicate taint findings which have different sources.
```

Comment:
I'm just a bit concerned that this could introduce duplicate findings for those that are not yet using `--dataflow-traces` like basically everybody using App.

Reply:
This is a good point 🤔. Maybe we can talk to Lewis about the expected behavior for CLI.

Reply:
talked to Lewis, and we believe that the best way to proceed is to duplicate findings in all cases. This will better the user experience (no fixing a finding, and then finding out there's still a finding), as well as being more in line with the JSON output.